src/init/VarResolver.ts

```diff
@@ -0,0 +1,139 @@
+import yargs from 'yargs';
```

Comment:
Would put this (and all classes that need separate files) in `src/init/variables/`.